test/clientTest.js

```diff
@@ -41,31 +38,31 @@ mc.supportedVersions.forEach(function (supportedVersion, i) {
       })
     })
 
-    describe('offline', function () {
+    describe('Offline mode tests', function () {
       before(function (done) {
-        console.log(new Date() + 'starting server ' + version.minecraftVersion)
+        console.log(`      Starting minecraft server for ${version.minecraftVersion}`)
```

Comment:
Please leave the date, it is useful to see what took time

Reply:
If you want to know the startup time, then extract it from the server log and print it with the server started text.

Reply:
How is that better ?
This is 100% reliable even if the minecraft wrap or the vanilla server have bugs (which is not that uncommon)

Reply:
Then use start and end times and get the difference. Printing random dates to console makes things hard to read.

Reply:
Well it's not random it's the continuity of all the server log printing the time. But yes we could also print the difference